python/multiaccountmetrics/multi_account_metrics.py

```diff
@@ -0,0 +1,95 @@
+import datetime
+from datetime.tz import tzlocal
+
+import boto3
+import botocore
+
+# You could use this code to load the instances from SSM parameter store.
+# import json
+# import os
+# ssm = boto3.client("SSM")
+# ROLE_INSTANCES_MAP = json.loads(
+#     ssm.get_parameter(
+#         Name=os.getenv('PARAMETER_NAME')
+#     )['Parameter']['Value']
+# )
+
+# Mapping of roles (accounts) to connect instances
+
+ROLE_INSTANCES_MAP = {
+    "arn:aws:iam::ACCOUNT:role/metrics-role": [
```

Comment:
I would put "ACCOUNT" and "metrics-role" in between the < > to make it obvious it should be replaced

Reply:
good call, done!